robot-core/src/main/resources/log4j.properties

```diff
@@ -3,4 +3,4 @@ log4j.appender.console.layout=org.apache.log4j.PatternLayout
 log4j.appender.console.layout.ConversionPattern=%-5p %m%n
 #log4j.appender.console.layout.ConversionPattern=%d [%t] %-5p %c - %m%n
 log4j.logger.user=DEBUG
-log4j.rootLogger=ERROR, console
+log4j.rootLogger=INFO, console
```

Comment:
Please leave the default logging level at ERROR. In #107 I added verbose flags, if you need them.